lib-api/src/main/java/org/ergoplatform/appkit/ErgoProver.java

```diff
@@ -63,8 +66,16 @@
      */
     SignedTransaction sign(UnsignedTransaction tx, int baseCost);
 
+    byte[] signMessage(SigmaBoolean sigmaTree, byte[] message, HintsBag hintsBag);
```

Comment:
Every public method need full documentation including: 1) description 2) all parameter 3) return value. 
In general, the description shouldn't just repeat what is obvious from the signature, instead it should complement it with special considerations derived from the essence of the method (something you would want to read, it is were not your method)